companion/lib/Service/Satellite.js

```diff
@@ -13,8 +13,13 @@ import LogController from '../Log/Controller.js'
  * 1.4.0 - Add TEXT_STYLE as ADD-DEVICE flags with FONT_SIZE as KEY-STATE property
  * 1.5.0 - Specify desired bitmap size with BITMAPS parameter when adding device
  * 1.5.1 - Remove surface size limit
+ * 1.6.0 - Allow for row/column notation,
+ * 		 - add streaming of text color when color is requested,
+ * 		 - allow choice of returned color format
+ * 		 - compatibility with internal CSS colors,
+ * 		 - allow buttons > 32
```

Comment:
does this actually change this?
I'm pretty sure that 1.5.1 already does this, but does keep using the same index system. But the indexes used in this api are always relative to the size of the grid it reported when adding the device, not the usual 1-32 that companion used to use internally

Reply:
There was still a check to abort if key number > LEGACY_MAX_BUTTONS
That's why I picked this up, I'm using the satellite API in a current project.

Reply:
Oh right yeah I can see that now.
I guess I didn't test that change very well..